### ISSUE TITLE
Startup scripts not able to pass more weblogic definitions to AdminServer

### ISSUE THREAD

**Issue by adamjk-dev** [biemond/biemond-orautils#21]
So, in the templates for the scripts that end up in /opt/scripts/wls, we are not able to append new options to the AdminServer startup script. It would be nice to have a way to pass in extra JVM arguments to the lines below.

https://github.com/biemond/biemond-orautils/blob/master/templates/startWeblogicAdmin.sh.erb#L12-L16

In our use case, we want to add the following to the script:
-Dweblogic.security.SSL.protocolVersion=TLS1 -Dweblogic.security.SSL.minimumProtocolVersion=TLSV1

**Comment by biemond** [comment]
nice, will do

**Comment by adamjk-dev** [comment]
Yeah, may want to expand this for both the nodemanager and admin server I assume. Then, can be generic to whatever extra arguments people need to pass in.

**Comment by adamjk-dev** [comment]
So, now that I got further along, I don't think this is necessary right? Without these params, I am still able to execute the scripts in /opt/scripts/wls and start/stop anything and those extra_args show up (so maybe they are picking them up from the startup.properties or something?). 

I can run the start/stop scripts in this directory without needing the extra_params to set TLS settings etc. It is working without them in these scripts.

**Comment by biemond** [comment]
yeah, everything goes through the nodemanager or the adminserver and that should pick up the arguments from the configuration

**Comment by adamjk-dev** [comment]
Gotcha, I think we can mark this as closed then.

**Comment by adamjk-dev** [comment]
Actually, wouldn't these params need to be set on the nodemanager start/stop scripts? Since they run startNodeManager.sh and don't take in the extra_args.

I don't think anything needs to change for the start/stop WeblogicAdmin.sh scripts, but do we need to add this notion of extra_args to start/stop nodemanager scripts?

**Comment by biemond** [comment]
Don't know if this parameter change the security behaviour between the secure node to admin communication then it should else not.
